Portfolio/index.html

```diff
@@ -18,7 +18,7 @@ <h1>Develop</h1>

```

Comment:
Consider adding alt attributes to provide text descriptions for the images for accessibility purposes.